simulatedClient/original/simulatedClient.py

```
@@ -0,0 +1,74 @@
+        asp_uid = str(uuid.uuid4())
+
+        # Send message to register the worker.
+        await websocket.send(json.dumps({
+            "code": MSG_TYPE["REGISTER"],
+            "client": True,
+            "asp_uid": asp_uid
+        }))
+
+        time.sleep(2)
+        try:
+            async for message in websocket:
```

Ruff (0.8.2)

Loop control variable `message` not used within loop body

Rename unused `message` to `_message`

(B007)